pkgs/applications/window-managers/hyprwm/hyprland/plugins.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, callPackage
+, pkg-config
+, gcc13Stdenv
+, hyprland
+}:
+let
+  mkHyprlandPlugin =
+    a@{ pluginName, ... }:
+    gcc13Stdenv.mkDerivation (a // {
+      pname = "${pluginName}";
+      nativeBuildInputs = a.nativeBuildInputs or [ ];
+      buildInputs = [ hyprland ]
+        ++ hyprland.buildInputs
+        ++ (a.buildInputs or [ ]);
+      meta = a.meta // {
+        description = a.meta.description ++
+          "\n\nPlugins can be installed via a plugin entry in the Hyprland NixOS or Home Manager options.";
+      };
+    });
+
+  plugins = {
+    hy3 = { fetchFromGitHub, cmake, hyprland }:
+      mkHyprlandPlugin {
+        pluginName = "hy3";
+        version = hyprland.version;
+
+        src = fetchFromGitHub {
+          owner = "outfoxxed";
+          repo = "hy3";
+          rev = "hl${hyprland.version}";
+          hash = "sha256-j49bEOLjBa1CH2gTwM+A2Edrw/GspE2m8q1teAn6SuQ=";
+        };
+
+        nativeBuildInputs = [ cmake pkg-config ];
```

Comment:
Shouldn't pkg-config be moved in the wrapper above to match hyperland?

Reply:
you were right fixed